cloud/azure/templates/azure_saml_ses/main.tf

```diff
@@ -37,16 +38,7 @@ module "app" {
   staging_ti_notification_mailing_list            = var.staging_ti_notification_mailing_list
   staging_applicant_notification_mailing_list     = var.staging_applicant_notification_mailing_list
 
-  civiform_time_zone_id = var.civiform_time_zone_id
-
-  civic_entity_small_logo_url = var.civic_entity_small_logo_url
-
-  adfs_admin_group = var.adfs_admin_group
-
-  login_radius_api_key       = var.login_radius_api_key
-  login_radius_metadata_uri  = var.login_radius_metadata_uri
-  login_radius_saml_app_name = var.login_radius_saml_app_name
-  saml_keystore_filename     = module.saml_keystore.filename
```

Comment:
**question:** Why do we want to remove these?

Reply:
Including these variables throw an error, removing them doesn't seem to harm the deployment so far, but I'm also unsure of the importance of these settings. I'll comment them out for now so the setup script runs and come back and fix it as a bug in a separate PR.

Reply:
Ah these are application variables. We can remove them since they shouldn't need to be set in terraform, but we should test that changing them in the config.sh file then updates them in the settings page:
<img width="1367" alt="Screenshot 2024-08-19 at 11 58 27 AM" src="https://github.com/user-attachments/assets/3b788611-06b4-4a6e-879b-145bcb84d551">

Can we also add in the description that we removed server application variables from the terraform script?

Reply:
done! will test the variable setting once staging is green and if it's not working i'll open a separate issue to track the fix

Reply:
Just came here to see how it went with the server vars! These are indeed safe to be removed and should be removed.